pom.xml

```
@@ -58,7 +58,7 @@
         <dependency>
             <groupId>io.jenkins</groupId>
             <artifactId>configuration-as-code</artifactId>
-            <version>1.0</version>
+            <version>1.32</version>
```

I suggest bumping to 1.42 as ordinal was fixed in 1.41 and secret resolver was improved in 1.42
```suggestion
            <version>1.42</version>
```

Parent pom should be bumped as well